Ginger/Ginger/Environments/GingerOpsEnvWizardLib/GingerOpsApplicationPage.xaml

```diff
@@ -1,19 +1,19 @@
-﻿<Page x:Class="Ginger.Environments.GingerAnalyticsEnvWizardLib.GingerAnalyticsApplicationPage"
+﻿<Page x:Class="Ginger.Environments.GingerOpsEnvWizardLib.GingerOpsApplicationPage"
       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
-      xmlns:local="clr-namespace:Ginger.Environments.GingerAnalyticsEnvWizardLib"
+      xmlns:local="clr-namespace:Ginger.Environments.GingerOpsEnvWizardLib"
       xmlns:GingerCore="clr-namespace:GingerCore;assembly=GingerCore"
       xmlns:Ginger="clr-namespace:Ginger" 
       mc:Ignorable="d" 
       d:DesignHeight="450" d:DesignWidth="800"
-      Title="GingerAnalyticsApplicationPage">
+      Title="GingerOpsApplicationPage">
 
     <Grid Background="{StaticResource $BackgroundColor_White}">
 
         <Ginger:ucGrid x:Name="SelectApplicationGrid" IsReadOnly="true" 
                       ShowTitle="Collapsed" ShowRefresh="Collapsed" ShowEdit="Collapsed" ShowAdd="Collapsed" ShowClearAll="Collapsed" ShowDelete="Collapsed" ShowUpDown="Collapsed"/>
-
+        <Label x:Name="xNote" Content="Note: Environment will only be imported on Clicking Finish" HorizontalAlignment="Left" VerticalAlignment="Bottom" Margin="10,0,0,10" />
```

Comment:
Also add note about saving modified solution file

Reply:
added the note, do I need to put break ? Because right now on laptop screen we need to expand the window to see the complete message.